home.css

```diff
@@ -31,6 +31,11 @@
         grid-template-columns: 1fr;
         grid-gap: 0.5rem;
```

Comment:
You can use px here since it is generally undesirable to scale a very small value in rem, but if you have a purpose behind usiing px, then no problem.